code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -77,15 +77,17 @@ GLOBAL_LIST_INIT(hardcoded_gases, list("o2","n2","co2","plasma")) //the main fou
 	moles_visible = 1
 	dangerous = TRUE
 
-/datum/gas/oxygen_agent_b
```

Comment:
Why axe agent B? It's a great mememos gas, and makes all sorts of meme maxcaps possible.

Reply:
It's not available in game and hasn't been for a several years.

Reply:
Lemme inform you of a beautiful thing known to the common tider as ✨ａｄｍｉｎｂｕｓ✨

Reply:
maybe give its properties to other gases?

Reply:
freon has 400 toobad its non reactive

Reply:
agent b is pointless bloat

Reply:
there will be no lack of meme gases to go autistic with, don't worry